libevm/VM.h

```diff
@@ -97,19 +101,17 @@ class VM: public VMFace
 	// space for memory
 	bytes m_mem;
 
-	// space for code and pointer to data
-	bytes m_codeSpace;
-	byte* m_code = nullptr;
+	// space for code
+	bytes m_code;
 
-	// space for stack and pointer to data
-	u256 m_stackSpace[1025];
-	u256* m_stack = m_stackSpace + 1;
-	ptrdiff_t stackSize() { return m_SP - m_stack; }
+	// space for data stack, grows towards smaller addresses from the end
+	u256 m_stack[1024];
```

Comment:
Since I'm not using the stack with any STL algorithms I agree that std::array is not very useful.  (It's funny, because I was a big advocate for these safe wrapper classes, and authored some of them, but I don't use them much.)